templates/webapps/galaxy/workflow/run_complete.mako

```diff
@@ -0,0 +1,55 @@
+<%inherit file="/base.mako"/>
```

Comment:
This file has been removed from ``dev`` - should it be removed here as well. Perhaps this needs a merge with latest dev.